cloud-chatops/README.md

```diff
@@ -120,3 +120,21 @@ The `secrets.json` file should look like the below and there is a template [here
   "GITHUB_TOKEN": "<your-token>"
 }
 ```
+
+### Automatic Container Update
+
+To pull the latest `docker-compose` file automatically, you can copy the contents of the cron_clone.sh file and create a new file in either the cron.hourly, cron.daily, cron.weekly, or cron.monthly directories. These directories can be found in `$HOME/etc/...`.
+
+Once created, you will need to change the file permissions accordingly:
+
+```shell
+  #Changing the file permissions.
+  chmod +x /etc/cron.hourly/<filename>
+
+  #You can test that the script is running correctly by using this command.
+  run-parts /etc/cron.hourly
+
+  #if you don't recieve any output, the script has not run.
+  ```
```

Comment:
If you try changing permisions while the file is in `/etc` you may need to use sudo. We should change the permissions inside the cloned repo before we copy the file over. This means we can avoid using sudo. 
e.g.
```suggestion
```shell
  # Clone the repo
  git clone https://github.com/stfc/cloud-docker-images.git

  # Change into repo directory
  cd cloud-docker-images/cloud-chatops

  # Change permissions on script
  chmod +x cron-clone.sh

  # Copy script to etc.hourly
  sudo cp cron-clone.sh /etc/cron.hourly/update_chatops

  # You can test that the script is running correctly by using this command.
  run-parts /etc/cron.hourly

  #if you don't recieve any output, the script has not run.
  ```
```

Reply:
Many thanks! Agreed with your suggestion around avoiding `sudo`, this makes sense as well copying the file rather than copying the contents. Avoids mistakes and makes it easier for whoever is deploying it.

Do we need `git clone https://github.com/stfc/cloud-docker-images.git` in this part of the README? I think if anyone is trying to automate this docker image, they will probably already have the repo. If they clone it again, might it risk having a duplicate repo?

Reply:
It's up to you. Like you say. They have probably already cloned the repo